js/pages/itemdetails.js

```diff
@@ -436,20 +436,87 @@ pageInit.itemdetails = function(category, assetId) {
 			})
 		}
 	}
-	
+
+	if(settings.itemdetails.showRevenue) {
+		const elem = html`
+		<div class="clearfix item-field-container">
+			<div class="text-label text-overflow field-label">Revenue</div>
+			<span class="icon-robux-16x16"> </span> <span class="field-content-revenue"></div>
+		</div>`
+
+		document.$watch(".field-content-sales", desc => {
+			desc.parentNode.after(elem)
+		})
```

Comment:
This won't work if showSales setting is disabled, as the field-content-sales element is never created.